core/src/main/java/discord4j/core/object/entity/Member.java

```diff
@@ -253,4 +255,22 @@ public String getNicknameMention() {
         return getServiceMediator().getRestClient().getGuildService()
                 .removeGuildMemberRole(guildId, getId().asLong(), roleId.asLong());
     }
+
+    /**
+     * Calculates the effective permissions of this member for a {@link GuildChannel}.
+     *
+     * @param c The guild channel for which to calculate permissions.
+     * @return A {@link Mono} where, upon successful completion, emits the {@link PermissionSet} of the effective
+     * permissions of this member for the channel. If an error is received, it is emitted through the {@code Mono}.
+     */
+    public Mono<PermissionSet> getEffectivePerms(GuildChannel c) {
```

Comment:
Same as comment in GuildChannel#getEffectivePerms
Additionally, can this just not delegate to that? `c.getEffectivePerms(this)`?

Reply:
I don't know if you saw my newest commit but I realized after submitting the PR that I could consolidate it into the one utility.